cfg/2.0.0/definitions.yaml

```diff
@@ -7988,17 +8122,17 @@ groups:
       tests:
         test_items:
         - flag: ""
```

Comment:
Are you sure about this, it seems to be testing an empty flag? Does this work to check that no results are returned?

Reply:
Yes, it checks that nothing is returned.
Without the eq, an empty flag would just catch anything